app/dbmodels.py

```diff
@@ -56,7 +56,7 @@ class Meta(BaseMeta):
         tablename = "server_data_distribution"
     id: int = ormar.Integer(primary_key=True)
     server_id: Server = ormar.ForeignKey(Server)
-    conversation_id: int = ormar.Integer()
+    conversation_id: = Conversation = ormar.ForeignKey(Conversation)
```

Comment:
```suggestion
    conversation_id:  Conversation = ormar.ForeignKey(Conversation)
```
Тут закралось лишнее равно))